plugins/org.eclipse.papyrus.umllight.core/resource/architecture/UMLLight.architecture

```diff
@@ -124,6 +124,24 @@
       <assistantRules xmi:type="gmfdiagrepresentation:AssistantRule" xmi:id="_vp8-8NuJEeiEnue7qWPHtw" permit="false"/>
       <palettes xmi:type="paletteconfiguration:PaletteConfiguration" href="platform:/plugin/org.eclipse.papyrus.umllight.core/resource/palette/UMLLightUseCaseDiagram.paletteconfiguration#/"/>
     </representationKinds>
+    <representationKinds xmi:type="gmfdiagrepresentation:PapyrusDiagram" xmi:id="_fPfAUNt4EeiKL9qM2SXqag" id="org.eclipse.papyrus.umllight.diagram.statemachine" name=" Light Statemachine Diagram" description="Simplified version of the statemachine diagram" icon="platform:/plugin/org.eclipse.papyrus.uml.diagram.statemachine/icons/obj16/Diagram_StateMachine.gif" concerns="__JaYANOxEeiTpIcrXWETSQ" implementationID="PapyrusUMLStateMachineDiagram" customStyle="platform:/plugin/org.eclipse.papyrus.umllight.core/resource/style/umllight_style.css" creationCommandClass="org.eclipse.papyrus.uml.diagram.statemachine.CreateStateMachineDiagramCommand">
```

Comment:
"State machine" is two words, so the name should be "Light State Machine Diagram".  And I'm not sure that it matters, but there's a space before " Light" in the name.